paper.tex

```diff
@@ -946,23 +960,27 @@ \subsection*{Illustrative examples}
 \begin{figure}
     \centering
     \includegraphics[width=0.8\textwidth]{figures/3-slices-whitespace.png}
-    \caption{Simulated ARG under Hudson coalescent (left). $N_e$ is constant
-    for each of the 3 time slices: $[(0:1000): 10 000, (1000:5000): 5000,
-    (5000:\infty):10 000]$. Remaining parameters: $L=10$kb, $20$ diploid samples,
-    $r=2.5e-8$, $148$ edges in ARG. A single $N_e$ value was estimated for each
-    slice of the ARG as defined by the true population size change time points.
+    \caption{Simulated ARG under the Hudson coalescent (left). $N_e$ is constant
+    for each of the 3 time slices:
+    from 0 to 1,000 generations ago $N_e$ is 10,000;
+    from 1,000 to 5,000 generations ago it is 5,000;
+    and past that it is 10,000.
+    Remaining parameters: genome length is 10kb, 20 diploid samples, and
+    $r=2.5\times10^{-8}$. 
+    There are 148 edges in the resulting ARG.
+    A single $N_e$ value was estimated for each
+    of the three slices of the ARG separated by
+    the true population size change time points.
     The inferred posterior distribution (right) using the SMC-likelihood was
-    obtained using a uniform prior. Number of edges per time slice
-    (from most recent slice to oldest): $[68, 57, 23]$. Locally unary nodes were
+    obtained using a uniform prior. The number of edges per time slice
+    (from most recent to oldest) was: 68, 57, and 23. Locally unary nodes were
     omitted for clarity.}
     \label{fig:3-arg-slices}
 \end{figure}
 
-Our final example concerns [TODO motivate this example; why is 
-this interesting/useful? Presumably we don't have to jointly estimate
- the ARG with demography, but can do so post-hoc? But, couldn't 
-you do that anyway if you had a full ARG using HKY?]
```

Comment:
I think it's an inherently interesting example. (a) it's a neat idea; and (b) it works pretty good. I don't really understand why the questions here are relevant.

Reply:
It's just not obvious to me how this is different to what you'd do with HKY. Can't you evaluate an abitrarily complex demography under HKY by basically keeping track of Ne etc as you go back through the events?

Reply:
Sorry do you mean KYF here? (and HKY is a model of nucleotide substitution?) And, I don't get what you're getting at, because the whole point here is something you can do without a full ARG - so, why a comparison to KYF? (if that's what you mean)

But - it is a good point that a better way to estimate Ne(t) could be to let  λ change with time in the likelihood itself. However, that would require optimizing a 3D function, instead of three 1D functions - so, it's kinda cool this works. You could imagine doing this in fine time slices in a large ARG to get Ne(t) on a fine scale (after testing). So - I still think this is a cool observation, using the fact this mostly factorizes in time slices.